src/collector/LogCollectCypressXhr.js

```diff
@@ -12,17 +12,20 @@ module.exports = class LogCollectCypressXhr {
   }
 
   register() {
-    const formatXhr = (options) =>
-      (options.renderProps.wentToOrigin ? '' : 'STUBBED ') +
-      options.consoleProps.Method + ' ' + options.consoleProps.URL;
+    // In Cypress 13+ this is under an extra props key
+    const consoleProps = (options) => options.consoleProps && options.consoleProps.props ? options.consoleProps.props : options.consoleProps
+
+    const formatXhr = (options) => (options.alias !== undefined ? '(' + options.alias + ') ' : '') +
+      (consoleProps(options)["Request went to origin?"] !== 'yes' ? 'STUBBED ' : '') +
```

Comment:
Just trying to help get the fix for the breaking tests identified -- Is the fact that `(options.renderProps.wentToOrigin ? '' : 'STUBBED ')` no longer present cause a break for Cypress < 12?

Also, completely blind question, so forgive my ignorance if it is showing -- anything in this project preventing the use of optional chain operators, i.e. instead of `options.consoleProps && options.consoleProps.props` you could have `options.consoleProps?.props`

Reply:
That could be one of the reasons indeed.

Optional chaining was introduced in node version 14. We currently require cypress > 10 for current version, but that doesn't require node >= 14 yet. If we would start using it we would need to declare that node >= 14 is required, in docs and in package.json. I would say leave it for now.

Reply:
Good shout, have reverted this change